### PR TITLE
feat(query): added "PSP With Unrestricted Access to Host Path" for Kubernetes

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- Added "PSP With Unrestricted Access to Host Path" for Kubernetes

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/psp_with_unrestricted_access_to_host_path/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "de4421f1-4e35-43b4-9783-737dd4e4a47e",
+  "queryName": "PSP With Unrestricted Access to Host Path",
+  "severity": "MEDIUM",
+  "category": "Access Control",
```
Comment (JoaoDuarteGomes) [inline assets/queries/k8s/psp_with_unrestricted_access_to_host_path/metadata.json:5]: This should be categorized as "Resource Management".

File: assets/queries/k8s/psp_with_unrestricted_access_to_host_path/metadata.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "de4421f1-4e35-43b4-9783-737dd4e4a47e",
+  "queryName": "PSP With Unrestricted Access to Host Path",
+  "severity": "MEDIUM",
```
Comment (JoaoDuarteGomes) [inline assets/queries/k8s/psp_with_unrestricted_access_to_host_path/metadata.json:4]: I would bump this up to High severity.

**Review by JoaoDuarteGomes — CHANGES_REQUESTED** [review]
The category needs to be changed, the severity is more of a recommendation.

**Review by JoaoDuarteGomes — APPROVED** [review]
Category and Severity LGTM.

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**